test/staged.jl

```diff
@@ -170,7 +170,7 @@ let gf_err, tsk = @async nothing # create a Task for yield to try to run
     end
     @test_throws ErrorException gf_err()
     @test_throws ErrorException gf_err()
-    @test gf_err_ref[] == 2
+    @test gf_err_ref[] == 4
```

Comment:
the actual value is implementation-defined, other than being non-zero and < 10000